### ISSUE TITLE
This resource is no longer available under v1 APIs

### ISSUE THREAD

**Issue by ruralcoder** [ArkeologeN/node-linkedin#93]
I am using your library from a Firebase function (firebase linked in sample from github)

After calling the 'me' method 

`linkedin.people.me(async (error, userResults)`

I am getting this object in userResults

```
{ errorCode: 0,
  message: 'This resource is no longer available under v1 APIs',
  requestId: 'T81DX5FAEW',
  status: 410,
  timestamp: 1566548902928 }
```

Expected user ID, name, photo, etc.

**Comment by dawogfather** [comment]
yeah it's trying to use v1 of the API which linkedin removed in March 2019 so I think you might need to try to use 
` npm install --save https://github.com/captainjackrana/node-linkedin`
and adjust accordingly. 
im doing it now actually so might make a PR for it
